ui/components/ErrorHandling.js

```diff
@@ -3,10 +3,12 @@ import { Cancel } from "@material-ui/icons";
 import { useSnackbar } from "notistack";
 
 /**
- * 
- * @param {Object} err 
- * @param {string} prefixMessage 
- * @param {("normal"|"fatal")} severity 
+ *  Show Snackbar when error occurs. Can be used in catch blocks
+ *  of functions.
+ *
+ * @param {Object} err
+ * @param {string} prefixMessage
+ * @param {("normal"|"fatal")} severity
```

Comment:
success, info, failure... ?

Reply:
@Abhishek-kumar09 @warunicorn19 are these standardized anywhere?

Reply:
I don't think they are.

Reply:
It is error severity, the snackbar is only for errors.

Reply:
yeah, it looks okay to me

Reply:
I explained on one of our last calls that there are three types of snackbar types: 

1. info. e.g. requested operation received and is in progress.
2. success. e.g. requested operation has concluded successfully.
3. failure. e.g. requested operation errored; failed to complete.

Reply:
Let's ensure that these (and other aspects of levels and types of notifications is captured in the Messaging Framework and Notification Center).

Reply:
MeshKit error severity levels: https://github.com/layer5io/meshkit/blob/fc8a197a4055c64dec7fe90a24750becab866404/errors/types.go#L16-L21

Reply:
Let's move forward with this PR until [the design spec](https://docs.google.com/document/d/1_zpEWBcC6ngOLen_E_nc-octiUdBbpmdwwIX6qzo70M/edit#) is fully-flushed out.